sdk/js/src/token_bridge/getForeignAsset.ts

```diff
@@ -203,3 +191,24 @@ export async function getForeignAssetNear(
   );
   return ret !== "" ? ret : null;
 }
+
+export async function getForeignAssetAptos(
+  client: AptosClient,
+  tokenBridgeAddress: string,
+  originChain: ChainId | ChainName,
+  originAddress: string
```

Comment:
I think the other methods all take Uint8Arrays, probably should make this consistent?

Reply:
Can you double check? I don't think any of the other methods in this file do that

Reply:
Upon updated review, Algorand and Near take (hex) strings, but don't require the `0x` prefix, so perhaps we should consider that.